README.md

```diff
@@ -27,7 +27,7 @@ Use this when the frontend WordPress site and CiviCRM are in the same wordpress
 1. Install and activate:
    - [Caldera Forms](https://wordpress.org/plugins/caldera-forms/) - WP plugin
    - [CiviCRM Form Processor](https://lab.civicrm.org/extensions/form-processor/) - CiviCRM extension
-   - [Caldera Forms integration with CiviCRM Form Processor](https://github.com/civimrf/cf-civicrm-formprocessor)  - this WP plugin
+   - [Caldera Forms integration with CiviCRM Form Processor](https://github.com/civimrf/cf-civicrm-formprocessor)  - this is a WP plugin
```

Comment:
True, it is 'a WP plugin', but the point of the comment is that it is 'this plugin'